Typo/0.10.5 fixes (#72)
Wrzlprnft/world-forge: data/map reveal.txt

outfit "Galaxy Map"
	category "Special"
	cost 0
	map 10000

> 	thumbnail "outfit/map"

Reviewer: It you're trying to match the order of the local map in the vanilla game, the thumbnail should come before the "map" attribute.

mission "Map Reveal"
	job
	repeat
	description "Reveal the entire map in one go! Accept this job and all vanilla systems and planets will be visited."

> 	source

Reviewer: The other missions in this plugin put the description after the source.

> 	description "Reveal the entire map in one go! Accept this job and all vanilla systems and planets will be visited."

Reviewer: This ought to be between `repeat` and `source`.

	on offer
		event "VISIT_EVERYTHING" 0
		fail

# Commented out lines indicate defined planets that are not assigned to any objects at the start of the game.

> # They cannot be visited and attempting to say they are renders this event invalid.

Reviewer: ```suggestion
# They cannot be visited and attempting to do so renders this event invalid.
```
I don't see anything wrong with this part.
